Multisig/UI/Settings/ImportOwnerViewController/OnboardingImportOwnerKeyViewController.swift

```diff
@@ -30,7 +30,8 @@ class OnboardingImportOwnerKeyViewController: UIViewController {
     }
 
     @objc private func didTapNextButton(_ sender: Any) {
-        // Here we need to show enter seedphase screen
```

Comment:
I would use factory like in the code above:
```
let vc = ViewControllerFactory.importOwnerViewController(presenter: self)
        present(vc, animated: true)
```

Reply:
This line will return the OnboardingImportOwnerKeyViewController